### PR TITLE
NR-53374 documentation for log configuration parsing rule

### PR DESCRIPTION
**Pull request by RavitejaSurampudi**
# Description

Please include a summary of the change and which issue is fixed (if relevant).

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

Please delete options that are not relevant.

- [x] My commit message follows [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] My code is formatted to [Go standards](https://go.dev/blog/gofmt)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes. Go [here](https://github.com/newrelic/terraform-provider-newrelic#testing) for instructions on running tests locally.

## How to test this change?

Please describe how to test your changes. Include any relevant steps in the UI, HCL file(s), commands, etc

- Step 1
- Step 2
- etc

### CODE REVIEW

**Review by sanderblue — APPROVED** [review]

**Review by mbazhlekova — APPROVED** [review]

File: website/docs/r/log_parsing_rule.html.markdown [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+Use this resource to create, update and delete New Relic Log Parsing Rule.
+
+## Example Usage
+
+Use this example to create the log parse rule.
+```hcl
+
+resource "newrelic_log_parsing_rule" "foo"{
+    account_id  = 12345
```
Comment (kidk) [inline website/docs/r/log_parsing_rule.html.markdown:19]: No magic numbers, please remote the account_id. It should be optional anyway.

File: website/docs/r/log_parsing_rule.html.markdown [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+```
+
+## Additional Example
+Use this example to validate a grok pattern and create the log parse rule.  More
+information on grok pattern can be found [here](https://docs.newrelic.com/docs/logs/ui-data/parsing/#grok)
+```hcl
+data "newrelic_test_grok_pattern" "grok"{
+    account_id  = 12345
```
Comment (kidk) [inline website/docs/r/log_parsing_rule.html.markdown:35]: Same comment here, remove the account_id

File: website/docs/r/log_parsing_rule.html.markdown [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    grok        = "%%{IP:host_ip}"
+    log_lines   = ["host_ip: 43.3.120.2"]
+}
+resource "newrelic_log_parsing_rule" "foo"{
+    account_id  = 12345
```
Comment (kidk) [inline website/docs/r/log_parsing_rule.html.markdown:40]: Same comment here, remove the account_id

**Review by kidk — CHANGES_REQUESTED** [review]

File: website/docs/r/log_parsing_rule.html.markdown (hunk 1 above)
Comment (RavitejaSurampudi) [reply website/docs/r/log_parsing_rule.html.markdown]: removed account Id

File: website/docs/r/log_parsing_rule.html.markdown (hunk 2 above)
Comment (RavitejaSurampudi) [reply website/docs/r/log_parsing_rule.html.markdown]: removed account Id

File: website/docs/r/log_parsing_rule.html.markdown (hunk 3 above)
Comment (RavitejaSurampudi) [reply website/docs/r/log_parsing_rule.html.markdown]: removed account Id

**Pull request merged**